src/util/Point.cpp

```diff
@@ -1,5 +1,60 @@
-//
-// Created by Carolin on 02.04.2020.
-//
+/**
+ * @file   Character.cpp
+ * @author Carolin
+ * @date   02.04.2020 (creation)
+ * @brief  Definition of the point class.
+ */
 
 #include "Point.hpp"
+
+namespace spy::util {
+    Point::Point(int x, int y) : x(x), y(y) {}
+
+    int Point::getX() const {
+        return x;
+    }
+
+    void Point::setX(int xCord) {
+        Point::x = xCord;
+    }
+
+    int Point::getY() const {
+        return y;
+    }
+
+    void Point::setY(int yCord) {
+        Point::y = yCord;
+    }
+
+    void Point::setLocation(int xCoord, int yCoord) {
+        Point::x = xCoord;
+        Point::y = yCoord;
+    }
+
+    void Point::translate(int dx, int dy) {
+        setLocation(Point::x + dx, Point::y + dy);
+    }
+
+    bool Point::isValid() {
+        return Point::x >= 0 && Point::y >= 0;
+    }
```

Comment:
Ist die validierung hier richtig aufgehoben? Ich würde da eher ins Spielfeld sowas im Sinne von `bool field.isInside(Point p)` machen

Reply:
Ja, ist sinnvolller.

Reply:
Habe das mal in #25 dazugeschrieben